### PR TITLE
add Retrieving Config section

### PR DESCRIPTION
**Pull request by nadinequinr3**

### CODE REVIEW

File: content/en/platform/corda/5.0/deploying-operating/config/dynamic.md [hunk 1]
```diff
@@ -57,3 +60,24 @@ For example, if the REST API is exposed on `localhost`, to set fields in the [me
    ```
    {{% /tab %}}
    {{< /tabs >}}
+
+## Retrieving Current Configuration Values
+
+You can retrieve the current values of the fields in a particular configuration section, using the GET method of the <a href ="../../reference/rest-api/C5_OpenAPI.html#tag/Configuration-API/operation/get_config__section_">`/api/v1/config` endpoint</a>.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/deploying-operating/config/dynamic.md:66]:
```suggestion
You can retrieve the current values of the fields in a particular configuration section by using the GET method of the <a href ="../../reference/rest-api/C5_OpenAPI.html#tag/Configuration-API/operation/get_config__section_">`/api/v1/config` endpoint</a>.
```

File: content/en/platform/corda/5.0/deploying-operating/config/dynamic.md [hunk 2]
```diff
@@ -57,3 +60,24 @@ For example, if the REST API is exposed on `localhost`, to set fields in the [me
...
+
+For example, to retreive the fields in the [messaging]({{< relref "./fields/messaging.md" >}}) section using Bash with Curl or PowerShell:
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/deploying-operating/config/dynamic.md:68]:
```suggestion
For example, to retrieve the fields in the [messaging]({{< relref "./fields/messaging.md" >}}) section using Bash with Curl or PowerShell:
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Review by paulmoloneyr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @nadinequinr3

**Pull request merged**